web_poet/fields.py

```diff
@@ -172,13 +182,18 @@ def get_fields_dict(
 
 
 T = TypeVar("T")
+UnknownFieldActions = Literal["ignore", "warn", "raise"]
 
 
 # FIXME: type is ignored as a workaround for https://github.com/python/mypy/issues/3737
 # inference works properly if a non-default item_cls is passed; for dict
 # it's not working (return type is Any)
 async def item_from_fields(
-    obj, item_cls: Type[T] = dict, *, skip_nonitem_fields: bool = False  # type: ignore[assignment]
+    obj,
+    item_cls: Type[T] = dict,  # type: ignore[assignment]
+    *,
+    skip_nonitem_fields: bool = False,
+    field_names: Optional[Iterable[str]] = None,
```

Comment:
If an unknown field has been unintentionally passed, it'd error out with:

```
# omitted stack trace

File ~/dev/z/web-poet/web_poet/fields.py:235, in <dictcomp>(.0)
    233     if skip_nonitem_fields:
    234         field_names = _without_unsupported_field_names(item_cls, field_names)
--> 235 return item_cls(**{name: getattr(obj, name) for name in field_names})

AttributeError: 'SomePage' object has no attribute 'unknown_field'
```

I think this is descriptive enough and we don't need additional handling of unknown fields here. Handling it in the `.fields_to_extract` property method should be enough.